### PR TITLE
Amend for sklearn wkfl

### PR DESCRIPTION
**Pull request by lucyleeow**

### CODE REVIEW

**Comment by lucyleeow** [comment]
ping @agramfort

File: submissions/random_forest_100/estimator.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from sklearn.ensemble import RandomForestRegressor
+
+
+def get_estimator():
+    pipeline = RandomForestRegressor(
+        n_estimators=50, max_leaf_nodes=3, random_state=61)
+    return pipeline
```
Comment (agramfort) [inline submissions/random_forest_100/estimator.py:7]: don't call this pipeline plz

maybe just "reg = RandomForestRegressor(...)"

I find it misleading that you call it pipeline.

File: submissions/starting_kit/estimator.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from sklearn.ensemble import RandomForestRegressor
+
+
+def get_estimator():
+    pipeline = RandomForestRegressor(
+        n_estimators=2, max_leaf_nodes=2, random_state=61)
```
Comment (agramfort) [inline submissions/starting_kit/estimator.py:6]: reg = RandomForestRegressor(...)?

**Comment by agramfort** [comment]
https://travis-ci.org/github/ramp-kits/boston_housing/jobs/676211098

**Comment by agramfort** [comment]
thanks @lucyleeow !

**Comment by agramfort** [comment]
can you deploy to server? link on top of notebook is:

http://www.ramp.studio/problems/boston_housing

also see that readme.md is not super up to date. thanks a lot @lucyleeow !

**Comment by agramfort** [comment]
this link:  https://ramp.studio/problems/boston_housing works though

**Pull request merged**